scripts/get-mq-queue-manager-web-url.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eE
+
+KUBECONFIG=""
+DEBUGFILE="/tmp/get_mq_queue_manager_web_url.log"
+if [[ -e "${DEBUGFILE}" ]]; then
+    rm -f "${DEBUGFILE}"
+fi
+
+# jq reads from stdin
+function parse_input() {
+  eval "$(jq -r '@sh "export KUBECONFIG=\(.KUBECONFIG) MQQUEUEMANAGERNAME=\(.MQQUEUEMANAGERNAME) MQQUEUEMANAGERNAMESPACE=\(.MQQUEUEMANAGERNAMESPACE)"')"
```

Comment:
we need to remove this dependency on jq in websphere module and in this one as well, but this could be done in another PR